SwedbankPaySDK/Classes/SwedbankPayWebViewController.swift

```diff
@@ -169,21 +169,29 @@ extension SwedbankPayWebViewController : WKNavigationDelegate {
         }
     }
     
-    private func decidePolicyFor(url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+    private func decidePolicyFor(
+        navigationAction: WKNavigationAction,
+        url: URL,
+        decisionHandler: @escaping (WKNavigationActionPolicy) -> Void
+    ) {
         attemptOpenUniversalLink(url) { opened in
             if opened {
                 decisionHandler(.cancel)
             } else {
-                self.decidePolicyForNormalLink(url: url, decisionHandler: decisionHandler)
+                self.decidePolicyForNormalLink(
+                    navigationAction: navigationAction,
+                    url: url,
+                    decisionHandler: decisionHandler
+                )
             }
         }
     }
     
-    private func decidePolicyForNormalLink(url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+    private func decidePolicyForNormalLink(navigationAction: WKNavigationAction, url: URL, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 155 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)